composer.json

```diff
@@ -11,7 +11,7 @@
     }
   ],
   "require": {
-    "php": "^7.2",
+    "php": "^7.2|^8.0",
```

Comment:
Tried this with php 7.3.24 and it didn't work. So might be better to have this as
``` json
     "php": ">=7.2",
```

Reply:
To clarify: I couldn’t install upcloud-php-api on PHP 7.3 using composer as this rule won’t allow 7.3 to be used

Reply:
Hmmm... Seems to work. I don't know what the issue was earlier with my php environment.